ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/ReadSymbolsFromMobileStyle/ReadSymbolsFromMobileStyle.cpp

```diff
@@ -143,7 +143,7 @@ ReadSymbolsFromMobileStyle::ReadSymbolsFromMobileStyle(QObject* parent /* = null
     m_currentSymbol->setColor(m_currentColor);
 
     // request symbol swatch
-    connect(m_currentSymbol, &MultilayerPointSymbol::createSwatchCompleted, m_currentSymbol, [this](QUuid id, QImage img)
+    connect(m_currentSymbol, &MultilayerPointSymbol::createSwatchCompleted, m_currentSymbol, [this](QUuid id, const QImage& img)
```

Comment:
`QUuid id` should be const &

Reply:
https://developers.arcgis.com/qt/cpp/api-reference/esri-arcgisruntime-symbol.html#createSwatchCompleted

Reply:
there are several occurrences of this